executor/executor.go

```diff
@@ -28,10 +29,18 @@ type Mount struct {
 	Readonly bool
 }
 
+type WinSize struct {
+	Rows   uint32
+	Cols   uint32
+	Xpixel uint32
```

Comment:
how are *pixel used?

Reply:
Honestly not sure, they are defined in the kernel winsize struct, but afaict they are unused on Linux.  It seems intended for terminal width/height in pixels instead of characters, not sure if there are any platforms that actually use it though.  I didnt find much useful information googling, so it seems safe to remove.  They are unused for containerd pty.Resize call anyway.

Reply:
Maybe this is the cursor position. But not sure if we should allow it or if it is even possible in containerd.